cluster-up/cluster/k8s-provider-common.sh

```diff
@@ -187,6 +187,22 @@ function wait_for_aaq_ready() {
     fi
 }
 
+function configure_cpu_manager() {
+    if [ ${KUBEVIRT_CPU_MANAGER_POLICY} == "static" ]; then
+        for node in $($kubectl get nodes -l "node-role.kubernetes.io/worker" --no-headers -o custom-columns=":metadata.name" | tr -d '\r'); do
+            # FIXME Replace with kubelet config drop ins once all providers are using k8s >= 1.28
+            # https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/#kubelet-conf-d
+            $kubectl drain ${node}
+            $ssh ${node} -- sudo systemctl stop kubelet
+            # FIXME ${ssh} is broken when using HereDocs, fix and replace this mess if possible.
+            $ssh ${node} -- "sudo rm -f /var/lib/kubelet/cpu_manager_state && sudo echo -e 'cpuManagerPolicy: static\nkubeReserved:\n  cpu: \"1\"\n memory: \"1Gi\"\ncpuManagerPolicyOptions:\n  full-pcpus-only: \"true\"' | sudo tee -a /var/lib/kubelet/config.yaml && sudo sed -i 's/cpuManagerReconcilePeriod\:\ 0s/cpuManagerReconcilePeriod\:\ 5s/g' /var/lib/kubelet/config.yaml"
```

Comment:
Is there any documentation for these values? Did you take them from docs or best practices?